biogtr/models/global_tracking_transformer.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         embedding_meta: dict = None,
         return_embedding: bool = False,
         decoder_self_attn: bool = False,
+        **kwargs,
```

Comment:
Should be fine, but let's come back to this - maybe we don't want to use kwargs here.

Reply:
keep as is for now may change later